LayoutTests/editing/text-iterator/sequential-collapsed-ranges.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
For new tests, we generally prefer using `js-test.js` instead of the pre/post scripts.